http.go

```diff
@@ -171,6 +204,29 @@ func NewReverseProxy(config *Config, db *sql.DB) *httputil.ReverseProxy {
 			}
 		}
 
+		// Check if this is a request we need to set extra fees for
+		extraFee := getExtraFee(postgres, apiKey)
+		if extraFee != nil && req.Method == "POST" {
+			switch originalPath {
+			case "/v2/swap/submarine", "/v2/swap/reverse", "/v2/swap/chain":
```

Comment:
Couldn't we switch this around? as we only need to get the partner fees for these 3 POST paths